fluentui_listitem/src/main/java/com/microsoft/fluentui/tokenized/tabItem/TabItem.kt

```diff
@@ -209,6 +225,22 @@ fun TabItem(
                     style = TextStyle(color = textColor, textAlign = TextAlign.Center)
                 )
             }
+            Spacer(modifier = Modifier.height(3.5.dp))
+
+                AnimatedVisibility(
+                    visible = selected,
+                    enter = fadeIn(animationSpec = tween(durationMillis = 300))+ expandHorizontally(animationSpec = tween(durationMillis = 300)),
+                    exit = fadeOut(animationSpec = tween(durationMillis = 300))+ shrinkHorizontally(animationSpec = tween(durationMillis = 300)),
+                )
+                {
+                    Box(
+                        modifier = Modifier
+                            .height(2.dp)
+                            .width(FluentGlobalTokens.SizeTokens.Size160.value)
+                            .background(shape = RoundedCornerShape(FluentGlobalTokens.CornerRadiusTokens.CornerRadiusCircle.value), color = textColor)
```

Comment:
Move fluentGlobaltokens into a variable and use here

Reply:
created the variable for it , done.